desired_resolution:
  x: 224
  y: 224

timing:
  camera: 0.025
  network_stage1: 0.025
  network_stage2: 0.0025
  network_stage3: 0.0025
  communication_stage1: 0.025
  communication_stage2: 0.025
  communication_stage3: 0.05

score_threshold: 0.3
desired_class: 15
aspect_ratio: 1.333333333

variance:
  x:
    min: .00387
    max: .00347
  y:
    min: .00144
    max: .00452
